deploy/operator/ztp/deploy_spoke_cluster.sh

```diff
@@ -61,3 +61,13 @@ echo "All ${CONTROL_PLANE_COUNT} agents have joined!"
 
 wait_for_condition "agentclusterinstall/${ASSISTED_AGENT_CLUSTER_INSTALL_NAME}" "Completed" "60m" "${ASSISTED_NAMESPACE}"
 echo "Cluster has been installed successfully!"
+
+installed=$(oc get -n ${ASSISTED_NAMESPACE} clusterdeployment/${ASSISTED_CLUSTER_DEPLOYMENT_NAME} \
```

Comment:
Could you use `wait_for_condition`?

Reply:
no, first thing I checked
it's under spec, not on conditions

Reply:
This is not a condition so we cannot use `wait_for_condition`, but we should have some retries and not fail immediately.

Reply:
+1 for retry

Reply:
done